src/mmw/apps/modeling/views.py

```diff
@@ -202,13 +207,29 @@ def _initiate_tr55_job_chain(model_input, job_id):
 
 @decorators.api_view(['GET'])
 @decorators.permission_classes((AllowAny, ))
-def district(request, id=None, state=None):
-    if id:  # query by unique id
-        district = get_object_or_404(District, id=id)
-        geojson = json.loads(district.polygon.geojson)
+def boundary_layers(request, table_id=None, obj_id=None):
+    if not table_id or not obj_id:
+        tiler_prefix = '//'
+        tiler_host = settings.MMW_TILER_HOST
+        tiler_port = settings.MMW_TILER_PORT
+        tiler_postfix = '/{z}/{x}/{y}'
+        tiler_base = '%s%s:%s' % (tiler_prefix, tiler_host, tiler_port)
+
+        def augment(index, dictionary):
+            retval = {}
+            retval['display'] = dictionary['display']
+            retval['tableId'] = index
+            retval['endpoint'] = urljoin(tiler_base, index + tiler_postfix)
+            return retval
+
+        layers = [augment(i, d)
+                  for i, d in settings.BOUNDARY_LAYERS.items()]
+        return Response(layers)
+    else:
+        table_name = settings.BOUNDARY_LAYERS[table_id]['table_name']
+        cursor = connection.cursor()
+        query = 'SELECT geom FROM %s WHERE id = %s' % (table_name, obj_id)
```

Comment:
Consider passing `obj_id` as a parameter to the query to avoid any possibility of a SQL injection. Ex. `cursor.execute(query, (obj_id,))` instead of string interpolation. The `table_name` value comes from a whitelist so I don't see any risk there.